docs/resources/Poll.md

```diff
@@ -0,0 +1,138 @@
+| layout_type?      | integer                                                                                             | The [layout type](#DOCS_RESOURCES_POLL/layout-type) of the poll. Defaults to... DEFAULT! |
+
+### Layout Type
+
+We might have different layouts for polls in the future.
+For now though, this number will be 1.
+
+| Type    | ID | Description                    |
+|---------|----|--------------------------------|
+| DEFAULT | 1  | The, uhm, default layout type. |
+
+### Poll Media Object
+
+The poll media object is a common object that backs both the question and answers.
+The intention is that it allows us to extensibly add new ways to display things in the future.
+For now, `question` only supports `text`, while answers can have an optional `emoji`.
+
+###### Poll Media Object Structure
+
+| Field  | Type                                                | Description            |
+|--------|-----------------------------------------------------|------------------------|
+| text?  | string                                              | The text of the field  |
+| emoji? | partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) | The emoji of the field |
+
+`text` should always be non-null for both questions and answers, but please do not depend on that in the future.
+The maximum length of `text` is 300 for the question, and 55 for any answer.
+
+When creating a poll answer with an emoji, one only needs to send either the `id` (custom emoji) or `name` (default emoji) as the only field.
+
+### Poll Answer Object
+
+The `answer_id` is a number that labels each answer.
+As an implementation detail, it currently starts at 1 for the first answer and goes up sequentially.
+We recommend against depending on this sequence.
+
+Currently, there is a maximum of 10 answers per poll.
+
+###### Poll Answer Object Structure
+
+| Field       | Type                                                                                    | Description            |
+|-------------|-----------------------------------------------------------------------------------------|------------------------|
+| answer_id\* | integer                                                                                 | The ID of the answer   |
+| poll_media  | [Poll Media Object](#DOCS_RESOURCES_POLL/poll-media-object-poll-media-object-structure) | The data of the answer |
+
+\* Only sent as part of responses from Discord's API/Gateway.
+
+### Poll Results Object
+
+In a nutshell, this contains the number of votes for each answer.
+
+Due to the intricacies of counting at scale, while a poll is in progress the results may not be perfectly accurate.
+They usually are accurate, and shouldn't deviate significantly -- it's just difficult to make guarantees.
+
+To compensate for this, after a poll is finished there is a background job which performs a final, accurate tally of votes.
+This tally concludes once `is_finalized` is `true`.
```

Comment:
Is the tally following the flag, or is the flag following the tally? 
Currently it reads as the former to me.

Reply:
The flag follows the tally. Honestly I find this sentence clear that it is the latter. 🤔